uber/templates/signups/shirt_size.html

```diff
@@ -3,18 +3,56 @@
 {% block backlink %}{% endblock %}
 {% block content %}
 
-<h2> Tell Us Your Shirt Size </h2>
+<form method="post" action="shirt_size">
+{{ csrf_token() }}
 
-One of the perks of volunteering is a tshirt for anyone who takes at least 6 weighted hours worth of shifts.
+<h2> Tell Us Your Shirt Size </h2>
 
-<br/> <br/>
+<p>One of the perks of volunteering is a tshirt for anyone who takes at least 6 weighted hours worth of shifts.</p>
 
-<form method="post" action="shirt_size">
-{{ csrf_token() }}
-<select name="shirt">
-    {{ options(opts,attendee.shirt) }}
-</select>
-<input type="submit" value="Submit Size" />
+{% if attendee.gets_staff_shirt and c.SHIRTS_PER_STAFFER > 1 %}
```

Comment:
Note that one thing which is NOT handled here is staffers who are added past the shirt deadline.

In theory that doesn't happen because we can no longer add staffers.  In practice, that's something which regdesk and stops managers sometimes do past the deadline, which we discussed adding back in over this past weekend.

If we wanted, we could program in some logic both here an in our counts that disallows those people from collecting shirts.  In practice the number should be small enough that it should be okay to not worry about that (and we never have worried about that in the past anyway).